metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityMergeEngine.java

```diff
@@ -146,6 +148,14 @@ private static AgentType convertAndAddAgent(Agent agent, List<AgentType> destina
     // dateList
     agentType.setDateList(ItemExtractorUtils.extractLabelResources(agent.getDate(), Date::new));
 
+    // placeOfBirth
+    agentType.setPlaceOfBirthList(
```

Comment:
do you have a format standard (in a file that I can import) for the code ?

Reply:
We're using the google java format style. 
https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml

Reply:
Thanks